installation/scripts/testing-common.sh

```diff
@@ -152,6 +152,26 @@ function waitForTestPodsTermination() {
     return 1
 }
 
+function deleteCTS() {
+  local retry=0
+  local suiteName=$1
+
+  log "Deleting ClusterTestSuite ${suiteName}" nc bold
+  while [[ ${retry} -lt 20 ]]; do
+        kubectl delete cts ${suiteName}
```

Comment:
Doing this this way, we lose the info about what happened, what error occurred, please do it like this:
```bash
msg=$(kubectl foo 2>&1)
status=$?
```
Then check status, and log msg. In case of cts we might fail due to too long grace period (it is by default determined on the size of the resource). Might want to consider adding `--now=true --wait=true`

Reply:
there is no need to wait for the deletion and there is no need for "immediate shutdown". It is important to delete the CTS in a near future